### DISCUSSION TITLE
Rollback game version

### DISCUSSION THREAD

**Discussion by SPHINXUS** [wolveix/satisfactory-server#86] [Q&A] [answered]
Hey there,

Is it possible to specify/rollback the game version ?
I forgot to put the "skip update" option so it updated on its own, now our client version is incompatible and we don't want to risk breaking our save as it has happened before.

Keep up the good work !

**Comment by wolveix** [comment, marked as answer]
Hey!

Thanks :) This would not be a simple thing to implement, unfortunately. There are guides on how to do it using `SteamCMD`, so you _could_ achieve this yourself in this instance. Personally though, minor updates are very unlikely to break your factory. Of course major updates will (such as update 5), but not minor updates.

**Reply by SPHINXUS** [reply]
Thanks, I didn't think about steamcmd options. I'll try to tinker in your scripts then.
